data/COPYRIGHTS.md

```diff
@@ -0,0 +1,34 @@
+## Copyrights and Attribution
+
```

Comment:
I would add some comment at the very top that you only have to attribute the data you are actually using, which depends on the indicator you use/request.

Reply:
I added a note to the top of the page.